pkg/api/v1/types.go

```diff
@@ -846,17 +846,26 @@ type ExecAction struct {
 	Command []string `json:"command,omitempty"`
 }
 
-// Probe describes a liveness probe to be examined to the container.
+// Probe describes a health check probe to be examined to the container.
```

Comment:
This description needs to be reworded:

`Probe describes a check to be performed against a container to determine whether it is alive or ready to take traffic.`